Cargo.toml

```diff
@@ -98,3 +98,5 @@ opt-level = 3
 
 [workspace]
 members = ["crates/*"]
+exclude = ["examples/tcp_server"]
```

Comment:
why?

Reply:
I was getting failures in CI. We either exclude it or include it, but not defining it has triggered intermittent issues.